test-data/hash-unquoted-key.pl

```diff
@@ -0,0 +1,7 @@
+use strict;
+use warnings;
+
+use HTTP::Status ();
+
+my %foo;
+$foo{is_info} = 1;
```

Comment:
It looks like this line has a trailing newline error.

Reply:
Fixed